src/lib/validations/utils.ts

```diff
@@ -35,3 +40,34 @@ export function getMultilineStatement( statementRegex: RegExp ): ( line: string
 		return matchingStatements;
 	};
 }
+
+export function getFileType( filepath: string ): string {
+	// Check if the computer has the file command installed
+	try {
+		cp.execFileSync( '/usr/bin/file', [ '-v' ] );
```

Comment:
I believe this check could be better placed somewhere else where it would be shown outside of the loop. 
Maybe before it starts, what do you think? 
https://github.com/Automattic/vip-cli/pull/1970/files#diff-84484f13e45d85ea3750110cf1f1485773c64ec3efefe7278f3a6d720e26d9a7L62